examples/rest-react/src/mocks/handlers.js

```diff
@@ -2,7 +2,7 @@ import { rest } from 'msw'
 
 export const handlers = [
   rest.post('/login', (req, res, ctx) => {
-    const { username } = req.body
+    const { username } = JSON.parse(req.body)
```

Comment:
Let's use the `req.json()` method instead:

```suggestion
    const { username } = await req.json()
```

You can also update other examples that rely on `req.body` but actually intend to read it as JSON. This is a new API we've introduced recently. It shouldn't fail the examples as they aren't using the latest MSW so let's make sure we update `msw` as well with this change.